extensions/websockets-next/deployment/src/test/java/io/quarkus/websockets/next/test/utils/WSClient.java

```diff
@@ -122,7 +123,7 @@ public Buffer waitForNextMessage() {
     }
 
     public void waitForMessages(int count) {
-        Awaitility.await().until(() -> messages.size() >= count);
+        Awaitility.await().atMost(Duration.ofSeconds(10)).until(() -> messages.size() >= count);
```

Comment:
Hm, 10 seconds is not that long... esp. for slow machines. I think that we should use longer interval (20?) by default and maybe make it configurable?

Reply:
I think I have changed this one when I was thinking about 5 second timeout Bruno suggested for waiting for traces (it was in memory handler). I'd not do that if I realized it was method inside WSClient and not the code I was adding.

TL;DR; I can see in `org.awaitility.constraint.AtMostWaitConstraint` that default value when it is reset is 10 seconds so this change is unnecessary. I'll drop it, it's good opportunity to re-run CI again and see if tests are not flaky.